p2p/transport/quic/virtuallistener.go

```diff
@@ -68,6 +69,11 @@ func (r *acceptLoopRunner) RmAcceptForVersion(v quic.VersionNumber) {
 	r.muxerMu.Lock()
 	defer r.muxerMu.Unlock()
 
+	if r.muxerClosed {
+		// Already closed, all versions are removed
```

Comment:
Is this an error-specific closing that basically indicates a failed start? If so maybe document/name it like that.

If so, does this mean that `Accept` should be modified as well? Want to make sure we don't end up in a position where we can keep calling accept but not RmAccept.

Reply:
We see the situation that quic-go's `Accept` sometimes errors on Windows machines. It might or might not be related to https://github.com/quic-go/quic-go/issues/1737. I don't have a Windows machine, so I can't really debug this. Also, I care more about other OSes ;)

There should probably be some logging, but I'm fine adding this in a later PR.